tests/src/rules/named.js

```diff
@@ -244,14 +244,11 @@ ruleTester.run('named', rule, {
     //   }],
     // }),
 
-    // flow types
+    // should ignore imported flow types, even if they don’t exist
     test({
       code: 'import type { MissingType } from "./flowtypes"',
       parser: 'babel-eslint',
-      errors: [{
-        message: "MissingType not found in './flowtypes'",
-        type: 'Identifier',
-      }],
+      errors: [],
     }),
```

Comment:
Looking at the overall structure of the test, I suggest moving this into the group of valid examples, instead of having it under invalid but with no errors.

If we'll just ignore type imports, I'd even go a step further and remove the remaining valid `import type` examples since they convey the impression that the plugin still cares about whether a type import is valid or not.

Reply:
Good suggestions, I made it so.